libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-advanced-filters-dialog.html

```diff
@@ -0,0 +1,60 @@
+<div sds-dialog-title>Advanced Filters</div>
```

Comment:
what is this dialog..?

Reply:
This was a temporary file and I have removed it now